Sdk/Speckle.Connectors.Utils/Connector.cs

```diff
@@ -10,7 +10,7 @@ public static class Connector
 {
   public static IDisposable? Initialize(HostApplication application, HostAppVersion version)
   {
-    TypeLoader.Initialize(typeof(Base).Assembly, typeof(GSAAssembly).Assembly);
+    TypeLoader.Initialize(typeof(Base).Assembly, typeof(Point).Assembly);
```

Comment:
looks tangential to this pr right?

Reply:
Yes, sorry it became somewhat irrelevant, this is how we're registering which assemblies contain Base subtypes.

I would have needed to add Speckle.Connectors.Utils here (via a `typeof(Conversion Result).Assembly`

My change from GSAAssembly to Point was simply to make it easier to see that we're getting the Objects assembly